package.json

```diff
@@ -1826,6 +1827,7 @@
     "terser": "^5.34.0",
     "terser-webpack-plugin": "^4.2.3",
     "tough-cookie": "^5.0.0",
+    "trace": "^3.1.1",
```

Comment:
Both added as dev dependencies because, for now, we only want to use them in dev mode only.

Reply:
New dependencies!!! Owner needed & update in `rennovate`

Reply:
`trace` only gets a 50/100 [Package Health Score](https://snyk.io/advisor/npm-package/trace) and hasn't seen a commit in 6 years! `clarify` is fine. Have you looked into alternatives to adding an unmaintained package?
For example: native async/await (with recent v8 improvements), async-hooks, or [sentry](https://snyk.io/advisor/npm-package/@sentry/node)

Reply:
> `clarify` is fine.

It looks like `clarify` is [the same](https://snyk.io/advisor/npm-package/clarify) 😞 

And their only dependency, `stack-chain` is in a [similar situation](https://snyk.io/advisor/npm-package/stack-chain) (although slightly better, 65/100 since it doesn't have any dependencies).

> Have you looked into alternatives to adding an unmaintained package?

I couldn't find any other alternatives (maybe it's a search-skill problem, though 😅).

> For example: native async/await (with recent v8 improvements), async-hooks, something else?

IIUC, `async/await` is the solution, but given the size of Kibana, identifying where we are missing them is the problem, and that's what these deps can help with. 

Also, there are places where we can't/don't want to use `async/await`: Task Manager needs to run on a timer-based approach. So their promises are necessarily detached from the main call-stack.

Reply:
FWIW, I submitted a PR to [improve Node.js v20 support](https://github.com/AndreasMadsen/clarify/pull/14) in `clarify` and it was accepted, merged, and published pretty quickly (the health score is now [75/100](https://snyk.io/advisor/npm-package/clarify)). 

I wonder if the lack of maintenance in the packages is more of an indication that they don't need changes instead of a sign of bad maintenance.

Reply:
Amazing 🥇

Reply:
I added the deps to renovate in [`b7fd4d9` (#195916)](https://github.com/elastic/kibana/pull/195916/commits/b7fd4d96ff699cdc5621f4bab77213074f95cfde) in case future improvements are applied.